dumper.go

```diff
@@ -50,49 +60,154 @@ type fileInfo struct {
 	Ext  string
 }
 
-func dump(acmeFile, dumpPath string, crtInfo, keyInfo fileInfo, domainSubDir bool) error {
-	f, err := os.Open(acmeFile)
+// Backend represents a data source for ACME data
+type Backend string
+
+const (
+	// FILE backend
+	FILE Backend = "file"
+	// CONSUL backend
+	CONSUL Backend = "consul"
+	// ETCD backend
+	ETCD Backend = "etcd"
+	// ZK backend
+	ZK Backend = "zk"
+	// BOLTDB backend
+	BOLTDB Backend = "boltdb"
```

Comment:
Could you explain why you ar trying to create a pseudo enum?
Simple string constants seems enough.

Reply:
I do not code much with go, so basically I look on code dependency how different situations are relaized. In that specific case, I saw https://github.com/abronan/valkeyrie/blob/7828df188bfd293509402d9ca8d73bf439f2f8a8/store/store.go#L10-L27 . But if thats not best practice, I use simple strings.